docs/api/naj-quick-reference.md

```diff
@@ -45,15 +45,30 @@ const { keyStores } = nearAPI;
 const keyStore = new keyStores.BrowserLocalStorageKeyStore();
 ```
 
+<!--Using Credentials Directory-->
+
+```js
+// creates keyStore from credentials directory
+// you should have login the account with NEAR CLI: https://docs.near.org/docs/tools/near-cli#near-login
+
+const { keyStores } = nearAPI;
+const homedir = require("os").homedir();
+const CREDENTIALS_DIR = ".near-credentials";
+const credentialsPath = path.join(homedir, CREDENTIALS_DIR);
+const keyStore = new keyStores.UnencryptedFileSystemKeyStore(credentialsPath);
+```
+
 <!--Using a File-->
 
 ```js
 // creates keyStore from a provided file
 // you will need to pass the location of the .json key pair
 
-const { keyStores } = nearAPI;
-const KEY_PATH = "~./near-credentials/testnet/example-account.json";
-const keyStore = new keyStores.UnencryptedFileSystemKeyStore(KEY_PATH);
+const { keyStores, KeyPair } = nearAPI;
+const KEY_PATH = "~/.near-credentials/testnet/example-account.json";
+const credentials = JSON.parse(fs.readFileSync(KEY_PATH));
+const keyStore = new keyStores.InMemoryKeyStore();
+keyStore.setKey(networkId, appName, KeyPair.fromString(credentials.private_key));
```

Comment:
```suggestion
const { KeyPair, keyStores } = require("near-api-js");
const fs = require("fs");
const homedir = require("os").homedir();

const ACCOUNT_ID = "near-example.testnet";  // NEAR account tied to the keyPair
const NETWORK_ID = "testnet";
const KEY_PATH = "/.near-credentials/testnet/near-example.testnet.json";

const credentials = JSON.parse(fs.readFileSync(homedir + KEY_PATH));
const keyStore = new keyStores.InMemoryKeyStore();
keyStore.setKey(NETWORK_ID, ACCOUNT_ID, KeyPair.fromString(credentials.private_key));
```

Nice catch! Wasn't able to get my machine to read the json file by passing the KEY_PATH using '~' even though I am on a mac... so decided to include `homedir` in this example as well.

Reply:
ah, yes. that's right. thanks!